packages/fxa-graphql-api/.prettierrc

```diff
@@ -1,4 +1,7 @@
 {
   "printWidth": 100,
-  "singleQuote": true
+  "singleQuote": true,
+  "trailingComma": "none",
```

Comment:
Saw this in #5386 but looks like it came from over here, so copying the comment: We have `"trailingComma": "es5"` [in other `.prettierrc` configs](https://github.com/mozilla/fxa/search?q=%22trailingComma%22%3A+%22es5%22&unscoped_q=%22trailingComma%22%3A+%22es5%22) - probably want that here too?

Reply:
Do we? I've been having the commit hook go and drop them.